### PR TITLE
Fix filter's label mismatch

### PR DESCRIPTION
**Pull request by Or-Gabay**

### CODE REVIEW

File: src/main/treeDataProviders/dependenciesTree/dependenciesTreeNode.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export class DependenciesTreeNode extends vscode.TreeItem {
 
     public shallowClone(): DependenciesTreeNode {
         let clone: DependenciesTreeNode = new DependenciesTreeNode(this.generalInfo, this.collapsibleState);
+        clone.label = this.label;
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/dependenciesTreeNode.ts:99]: Only this line needed. Please revert all other changes.

Comment (Or-Gabay) [reply src/main/treeDataProviders/dependenciesTree/dependenciesTreeNode.ts:99]: Did you check it?

Comment (yahavi) [reply src/main/treeDataProviders/dependenciesTree/dependenciesTreeNode.ts:99]: yes

File: src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts [hunk 2]
```diff
@@ -15,9 +15,10 @@ export class MavenTreeNode extends DependenciesTreeNode {
         private _workspaceFolder: string,
         private _componentsToScan: Collections.Set<ComponentDetails>,
         private _treesManager: TreesManager,
-        parent?: DependenciesTreeNode
+        parent?: DependenciesTreeNode,
+        generalInfo?: GeneralInfo
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts:19]: We already have generalInfo in the parent
```suggestion
```

File: src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts [hunk 3]
```diff
@@ -15,9 +15,10 @@ export class MavenTreeNode extends DependenciesTreeNode {
...
     ) {
-        super(new GeneralInfo('', '', _workspaceFolder, ''), vscode.TreeItemCollapsibleState.None, parent);
+        super(generalInfo ?? new GeneralInfo('', '', _workspaceFolder, ''), vscode.TreeItemCollapsibleState.None, parent);
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts:21]:
```suggestion
        super(new GeneralInfo('', '', _workspaceFolder, ''), vscode.TreeItemCollapsibleState.None, parent);
```

File: src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts [hunk 4]
```diff
@@ -90,6 +91,23 @@ export class MavenTreeNode extends DependenciesTreeNode {
...
     }
 
+    /** @override */
+    public shallowClone(): MavenTreeNode {
+        const clone: MavenTreeNode = new MavenTreeNode(
+            this._workspaceFolder,
+            this._componentsToScan,
+            this._treesManager,
+            undefined,
+            this.generalInfo
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts:101]:
```suggestion
```

File: src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts [hunk 5]
```diff
@@ -90,6 +91,23 @@ export class MavenTreeNode extends DependenciesTreeNode {
...
+        );
+        clone.licenses = this.licenses;
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts:103]:
```suggestion
        clone.generalInfo = this.generalInfo;
        clone.licenses = this.licenses;
```

File: src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts (hunk 2 above)
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/mavenTreeNode.ts:19]:
```suggestion
```

**Pull request merged**